### ISSUE TITLE
[Component Page] 'Code' + 'Documentation' accordions don't have focus state on Safari

### ISSUE THREAD

**Issue by mollieru** [18F/web-design-standards#541]
*Steps to reproduce*
1. Navigate with a mouse on any component page
2. Click 'code' and/or 'documentation' accordion -- no focus state appears

http://screencast-o-matic.com/watch/coQoYhfdxX

*Expected behavior*
On click, accordion links should activate the focus state

*Test on *
OX 10.8
Safari 6.2

**Comment by carodew** [comment]
Duplicate of #534, which is more descriptive so we're going to keep that open and close this one.
